config/olm/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: openshift-marketplace
+namePrefix: openshift-builds-
+
+resources:
+- catalogsource.yaml
+
+images:
```

Comment:
This is not replacing the image for me.
Requires transformer config
```
images:
- kind: CatalogSource
  path: spec/image
```

Reply:
Done!